packages/react/src/experimental/index.ts

```diff
@@ -1,3 +1,6 @@
 'use client'
 
 export * from '../drafts'
+
+export {Stack, StackItem} from '../Stack'
+export type {StackProps, StackItemProps} from '../Stack'
```

Comment:
These should probably be in drafts because we export everything from drafts to experimental (but not the other way round)

Just until next major (when we can get rid of "drafts")